### PR TITLE
Preventing payload modification when sending message to child window

### PR DESCRIPTION
**Pull request by SidGulatiMsft**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description
When Teams Hub sends a payload to the child application (Viva Connection), we are unnecessarily wrapping it in an array.
This code path doesn't get executed for other apps like Viva Engage, and that is leading to a divergence in what payload is received by the app's callback method.
Through this PR, I am preventing the payload from getting modified.

### Main changes in the PR:

1. Prevent the payload from getting wrapped in an array before sending to the child frame

### Unit Tests added:

> Unit tests are required for all changes. If no unit tests were added as part of this change, please explain why they aren't necessary.

No.
This is specific to secure-broker and unit tests for the same can't be added here.

### End-to-end tests added:

No

### Change file added:

> Ensure the change file meets the [formatting requirements](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

Yes

### Next/remaining steps:

> List the next or remaining steps in implementing the overall feature in subsequent PRs (or is the feature 100% complete after this?).

- [ ] Make a corresponding change in Teams hub, to wrap the payload in an array
- [ ] Ensure that the downstream app (Viva Connections) can handle the incoming payload correctly

### CODE REVIEW

File: package-lock.json
Comment (TrevorJoelHarris) [inline package-lock.json:1]: 47,247 changes seems like too many changes. We don't currently have a package-lock.json file, why are you adding one?

File: change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (TrevorJoelHarris) [inline change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json:2]:
```suggestion
  "type": "minor",
```

File: change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "comment": "Preventing the payload for child frame to get wrapped in an array.",
```
Comment (TrevorJoelHarris) [inline change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json:3]:
```suggestion
  "comment": "Embedded apps no longer incorrectly get their parameters wrapped in an array",
```

**Review by TrevorJoelHarris — CHANGES_REQUESTED** [review]
Please make sure Graham also reviews this

**Comment by AE-MS** [comment]
There should be tests for this. @TrevorJoelHarris will/has opened a backlog item for our team to take care of that since it's a new class of tests and not something that would be reasonable for you to add as part of this PR.

**Review by AE-MS — APPROVED** [review]

File: package-lock.json
Comment (SidGulatiMsft) [reply package-lock.json:1]: removed

File: change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json (hunk 1 above)
Comment (SidGulatiMsft) [reply change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json]: done

File: change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json (hunk 2 above)
Comment (SidGulatiMsft) [reply change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json]: done

**Review by GrahamMcMynn — APPROVED** [review]

File: change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Embedded apps no longer incorrectly get their parameters wrapped in an array",
+  "packageName": "@microsoft/teams-js",
+  "email": "sigulati@microsoft.com",
+  "dependentChangeType": "minor"
```
Comment (SidGulatiMsft) [inline change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json:6]: @TrevorJoelHarris , I changed the "dependentChangeType" to minor. I am guessing that is ok?

**Review by TrevorJoelHarris — APPROVED** [review]

**Review by AE-MS — APPROVED** [review]

**Pull request merged**